banzai/dbs.py

```diff
@@ -411,7 +411,7 @@ def get_instruments_at_site(site, db_address=_DEFAULT_DB, ignore_schedulability=
 
 
 def get_master_calibration_image(image, calibration_type, master_selection_criteria,
-                                 realtime_reduction=False, db_address=_DEFAULT_DB):
+                                 use_older_calibrations=False, db_address=_DEFAULT_DB):
```

Comment:
I found the `use_older_calibrations` variable name a bit ambiguous. What about something like `use_only_past_calibrations` or something like that?